packages/komencikit/README.md

```diff
@@ -0,0 +1,31 @@
+# KomenciKit
+
+The KomenciKit library is a wrapper for the Komenci Service API, which is used for fee-less onboarding by allowing 
+the consumer to execute fee-less attestations with the help of MetaTransactions.
+
+The main actions that KomenciKit exposes are:
```

Comment:
Don't we need a wrapper for `escrow.withdraw`? This that's the only onboarding on-chain transaction not represented in this list.

And in general, must a transaction have a wrapper in KomenciKit in order to be subsidized?

Reply:
A transaction doesn't need to have a wrapper. I was actually inclined to remove some of the wrappers and rely on the `submitMetaTransaction` action that takes in an arbitrary MetaTransaction and pushes it. 
That will only work, however with transaction that we allow on the API side by filtering on contract address and methodID.

The only transaction that we actually _must_ wrap is the `Attestations.request` logic because there we're doing the actual subsidy batch and not just executing a simple meta transaction.